Lpd 49706 sharing modal

oliv-yu opened the pull request.

BryanEngler committed:
LPD-50745 portal-search-elasticsearch7-impl: Allow entryClassName as a sort field

https://liferay.atlassian.net/browse/LPD-50745

BryanEngler committed:
LPD-50745 portal-search-opensearch2-impl: Allow entryClassName as a sort field

https://liferay.atlassian.net/browse/LPD-50745

BryanEngler committed:
LPD-50745 portal-search-solr8-impl: Allow entryClassName as a sort field

https://liferay.atlassian.net/browse/LPD-50745

andrea-ale-sbarra committed:
LPD-51865 add custom fields support to CPConfigurationList entity

andrea-ale-sbarra committed:
LPD-51865 added requested filters to CPConfigurationList rest API

andrea-ale-sbarra committed:
LPD-51865 autogenerated code (gw buildREST)

rodrigoguedes committed:
LPD-50745 portal-search-solr8-impl: Add unit test for SearchSolrQueryAssemblerImpl

https://liferay.atlassian.net/browse/LPD-50745

rodrigoguedes committed:
LPD-50745 portal-search-elasticsearch7-impl: Add unit test for SortTranslator

https://liferay.atlassian.net/browse/LPD-50745

rodrigoguedes committed:
LPD-50745 portal-search-opensearch2-impl: Add unit test for SortTranslator

https://liferay.atlassian.net/browse/LPD-50745

oliv-yu committed:
LPD-49706 LPD-50637 site-cms-site-initializer: Create sharing modal content

oliv-yu committed:
LPD-49706 LPD-60112 site-cms-site-initializer: Update files dropdown to open share modal

oliv-yu committed:
LPD-49706 LPD-50637 site-cms-site-initializer: Add APIs and format data

oliv-yu committed:
LPD-49706 LPD-50637 portal-language-lang: Add language keys for share modal

oliv-yu committed:
LPD-49706 LPD-50637 site-cms-site-initializer: Updates to site initializer

add to property changes

Mikellorza committed:
LPD-62098 Change view history toolbar title as we have in the designs

oliv-yu committed:
LPD-49706 LPD-60111 site-cms-site-initializer: Keep original names of properties, add conditional success message

oliv-yu committed:
LPD-49706 LPD-60111 site-cms-site-initializer: Add validation for expirationDate

oliv-yu committed:
LPD-49706 LPD-50637 site-cms-site-initializer: Add a loading indicator + prevent accidental submission

adolfopa committed:
LPD-61564 Add new type column to depot entry

adolfopa committed:
LPD-61564 Autogenerated

# breaking

## What /modules/apps/depot/depot-api/src/main/java/com/liferay/depot/service/DepotEntryGroupRelLocalService.java

A new `type` parameter was added to the `getDepotEntryGroupRels` and `getDepotEntryGroupRelsCount` methods.

## Why

We need to differentiate between two different kinds of depots: spaces and asset libraries. This new parameter will allow to discriminate between these two types of depots.

----

# breaking

## What /modules/apps/depot/depot-api/src/main/java/com/liferay/depot/service/DepotEntryGroupRelService.java

A new `type` parameter was added to the `getDepotEntryGroupRels` and `getDepotEntryGroupRelsCount` methods.

## Why

We need to differentiate between two different kinds of depots: spaces and asset libraries. This new parameter will allow to discriminate between these two types of depots.

----

# breaking

## What /modules/apps/depot/depot-api/src/main/java/com/liferay/depot/service/DepotEntryLocalService.java

A new `type` parameter was added to the `addDepotEntry`, `getGroupConnectedDepotEntries` and `getGroupConnectedDepotEntriesCount` methods.

## Why

We need to differentiate between two different kinds of depots: spaces and asset libraries. This new parameter will allow to discriminate between these two types of depots.

----

# breaking

## What /modules/apps/depot/depot-api/src/main/java/com/liferay/depot/service/DepotEntryService.java

A new `type` parameter was added to the `addDepotEntry`, `getGroupConnectedDepotEntries` and `getGroupConnectedDepotEntriesCount` methods.

## Why

We need to differentiate between two different kinds of depots: spaces and asset libraries. This new parameter will allow to discriminate between these two types of depots.

----

manuelecastro committed:
LPD-61377 Add IllegalArgumentException and log errors

oliv-yu committed:
LPD-49706 LPD-60111 site-cms-site-initializer: Allow clearing expiration date

andrea-ale-sbarra committed:
LPD-51865 autogenerated code (gw buildService)

# breaking

## What modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPConfigurationListLocalServiceImpl.java

modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPConfigurationListLocalServiceImpl.java add addOrUpdate and update methods signature now expose ServiceContext

## Why

this change is required to support Custom Fields

----

# breaking

## What modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPConfigurationListServiceImpl.java

modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPConfigurationListServiceImpl.java add addOrUpdate and update methods signature now expose ServiceContext

## Why

this change is required to support Custom Fields

----

julienmouraddotcom committed:
LPD-58720 Add Support View Mode for Files

julienmouraddotcom committed:
LPD-58720 SF

oliv-yu committed:
LPD-49706 LPD-60112 site-cms-site-initializer: Apply share modal to web content

oliv-yu committed:
LPD-49706 LPD-50637 site-cms-site-initializer: Include some frontend tests

oliv-yu committed:
LPD-49706 LPD-50637 site-cms-site-initializer: Expiration date needs tooltip + apply 11:59pm cutoff

adolfopa committed:
LPD-61564 Preserve type when creating depot for staged group

adolfopa committed:
LPD-61564 Set type explicitly when creating depot entry

adolfopa committed:
LPD-61564 Replace usages

ambrinchaudhary committed:
LPD-58720 Move css to the module that is imported in the react component

ambrinchaudhary committed:
LPD-58720 Use a ClayLink

adolfopa committed:
LPD-61564 Add depot entry type to group rel

This will allow us to query the database using specific indexes and
avoid joins.

adolfopa committed:
LPD-61564 Add type constraint to finder

adolfopa committed:
LPD-61564 Add type to service methods

carolmariaabb committed:
LPD-61893 Show meaningful error messages when copying large image file into Rich Text object field and submitting the form

adolfopa committed:
LPD-61564 Replace usages

JavierMoral committed:
LPD-61906 Add scope to ERCInfoItemIdentifier

victorg1991 committed:
LPD-61783 Enable Scheduling in structures

felipelorenz committed:
LPD-60385 portal-search-opensearch2-impl: bump org.apache.httpcomponents.client5:httpclient5

https://liferay.atlassian.net/browse/LPD-60385

abfranca committed:
LRSD-10430 Fix swarm skills custom field got erased after linking a business event to a zendesk ticket

oliv-yu committed:
LPD-49706 LPD-50637 site-cms-site-initializer: Separate + move components, rename handlers

andrea-ale-sbarra committed:
LPD-51865 add custom fields support to CPConfigurationList entity (yaml file)

victorg1991 committed:
LPD-61783 Add expirationDate and reviewDate

victorg1991 committed:
LPD-61783 Add fields

victorg1991 committed:
LPD-61783 Add values

victorg1991 committed:
LPD-61783 Add test

adolfopa committed:
LPD-61564 Update and clean up integration tests

adolfopa committed:
LPD-61564 Integration tests

adolfopa committed:
LPD-61564 Upgrade process

victorg1991 committed:
LPD-61783 Add review and expiration date to the creator

victorg1991 committed:
LPD-61783 Add review and expired date to the update

A double update is needed since with the partialUpdate we cannot set the dates as null which is necessary

victorg1991 committed:
LPD-61783 Process dates

victorg1991 committed:
LPD-61783 Include null values if they are included in the request

victorg1991 committed:
LPD-61783 Update tests

DavysonMelo committed:
LPD-61893 Add playwright test

mxs2 committed:
Revert "LPD-56406 Escape Parameter URL in RenderStructureFieldMVCResourceCommand"

This reverts commit 9e0026c8aa444937a2bfd079bcca35ab3dd18f5a.

mxs2 committed:
LPD-62034 Properly escape definition parameter for DDMFormField

oliv-yu committed:
LPD-49706 LPD-50637 site-cms-site-initializer: Apply a CollaboratorService and update Collaborator

oliv-yu committed:
LPD-49706 LPD-50637 site-cms-site-initializer: No need to preventDefault, remove old collaboratorByTypeURLs

oliv-yu committed:
LPD-49706 LPD-50637 portal-language-lang: buildLang

oliv-yu committed:
LPD-49706 LPD-50637 site-cms-site-initializer: Use some constants, avoid hard-coding values

ling-alan-huang committed:
LPD-59878 Fix typo

ling-alan-huang committed:
LPD-59878 Fix typo

andrea-ale-sbarra committed:
LPD-51865 sf

victorg1991 committed:
LPD-61783 Fix TS

DavysonMelo committed:
LPD-61893 Add image used in upload test

JavierMoral committed:
LPD-61906 Add method that receives current group to be able to calculate the scopeExternalReferenceCode

Mikellorza committed:
LPD-62098 Add success messages for expire and restore

Mikellorza committed:
LPD-62098 buildLang

ambrinchaudhary committed:
LPD-58720 Add unit tests

ambrinchaudhary committed:
LPD-58720 Add functional test to assert that the View action is visible.

ambrinchaudhary committed:
LPD-58720 Hide View action for folders, they have their own View Folder action

brianikim committed:
LPD-62187 Use double value instead for simple formatting

brianikim committed:
LPD-62187 Functional Test

JhossephG committed:
LPD-61580 create postScopeScopeKeyByExternalReferenceCodeExpire

daiazius committed:
LPD-61377 Add test

JhossephG committed:
LPD-61580 methods implementation

interaminense committed:
LPD-60567 Follow Up, rename testids to fix playwright tests

JhossephG committed:
LPD-61580 add coverage

BryanEngler committed:
LPD-50745 portal-search-elasticsearch7-impl: Move translator to legacy package and refactor test

https://liferay.atlassian.net/browse/LPD-50745

BryanEngler committed:
LPD-50745 portal-search-opensearch2-impl: Explicitly set sort order and refactor test

https://liferay.atlassian.net/browse/LPD-50745

BryanEngler committed:
LPD-50745 portal-search-solr8-impl: Refactor test

https://liferay.atlassian.net/browse/LPD-50745

JhossephG committed:
LPD-61580 buildRest

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

oliv-yu commented:
ci:test:sf

liferay-continuous-integration commented:
Test suite sf has been triggered on http://test-1-34

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-34.liferay.com/job/test-portal-source-format/4223/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-source-format/builds/4223/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7d4645f3ec9fa04a5a7778bfbe084519851c3276">7d4645f3ec9fa04a5a7778bfbe084519851c3276</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/oliv-yu/liferay-portal/tree/LPD-49706-sharing-modal">LPD-49706-sharing-modal</a><br/>Branch GIT ID: <a href="https://github.com/oliv-yu/liferay-portal/commit/5e360e3c0271c08fced7a0f6bfb2b768b8c1be07">5e360e3c0271c08fced7a0f6bfb2b768b8c1be07</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-34.liferay.com/job/test-portal-source-format/4223/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-source-format/builds/4223/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-34.liferay.com/job/test-portal-source-format/4223">test-portal-source-format#4223</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-source-format/builds/4223/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/oliv-yu/liferay-portal/pull/106">oliv-yu#106</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/276196596">[master] ci:test:sf - oliv-yu &gt; oliv-yu - PR#106 - 2025-08-01[18:08:45]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>276196596</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-34.liferay.com/job/test-portal-source-format/4223/">test-portal-source-format#4223</a><br/></div></div>

oliv-yu commented:
ci:test:cms

liferay-continuous-integration commented:
Test suite cms has been triggered on http://test-1-28

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>cms</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/13478/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/13478/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:cms - 2 out of 5 jobs passed in 57 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7d4645f3ec9fa04a5a7778bfbe084519851c3276">7d4645f3ec9fa04a5a7778bfbe084519851c3276</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7d4645f3ec9fa04a5a7778bfbe084519851c3276">7d4645f3ec9fa04a5a7778bfbe084519851c3276</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/275975518">EE Development Acceptance (master) - 1560 - 2025-08-01[12:33:44]</a></p><details><summary><strong>ci:test:cms - 2 out of 5 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/13478/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>js-unit<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1293818/"><strike>js-unit/0/0</strike></a></li></ul></li><li>modules-integration-postgresql163<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1175084/"><strike>modules-integration-postgresql163/0/0</strike></a></li></ul></li><li>playwright-js-tomcat101-postgresql163<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1195105/">playwright-js-tomcat101-postgresql163/0/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1175077/">playwright-js-tomcat101-postgresql163/1/0</a></li></ul></li></ul></div><details><summary><strong>2 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/13478/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1293818//consoleText">js-unit/0/0</a><ol><li><details><summary><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1293818/testReport/(root)/:apps:site:site-cms-site-initializer:packageRunTest">:apps:site:site-cms-site-initializer:packageRunTest</a></summary><div><ol><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1293818//testReport/(root)/:apps:site:site-cms-site-initializer:packageRunTest/apps_site_site_cms_site_initializer_test_js_main_view_spaces_NewSpace_hasErrors_disables_the_Continue_button_when_the_name_field_is_empty">apps.site.site-cms-site-initializer.test.js.main_view.spaces.NewSpace hasErrors disables the Continue button when the name field is empty</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1293818//testReport/(root)/:apps:site:site-cms-site-initializer:packageRunTest/apps_site_site_cms_site_initializer_test_js_main_view_spaces_NewSpace_hasErrors_shows_error_message_when_space_name_is_more_than_150_characters_long">apps.site.site-cms-site-initializer.test.js.main_view.spaces.NewSpace hasErrors shows error message when space name is more than 150 characters long</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1293818//testReport/(root)/:apps:site:site-cms-site-initializer:packageRunTest/apps_site_site_cms_site_initializer_test_js_main_view_spaces_NewSpace_hasErrors_shows_error_message_when_space_name_is_numeric">apps.site.site-cms-site-initializer.test.js.main_view.spaces.NewSpace hasErrors shows error message when space name is numeric</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1293818//testReport/(root)/:apps:site:site-cms-site-initializer:packageRunTest/apps_site_site_cms_site_initializer_test_js_main_view_spaces_NewSpace_submits_form_with_custom_color">apps.site.site-cms-site-initializer.test.js.main_view.spaces.NewSpace submits form with custom color</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1293818//testReport/(root)/:apps:site:site-cms-site-initializer:packageRunTest/apps_site_site_cms_site_initializer_test_js_main_view_spaces_SpaceGeneralSettings_Errors_does_not_save_the_name_field_when_there_is_an_error_and_the_field_is_focused">apps.site.site-cms-site-initializer.test.js.main_view.spaces.SpaceGeneralSettings Errors does not save the name field when there is an error and the field is focused</a></li><li>...</li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1175084//consoleText">modules-integration-postgresql163/0/0</a><ol><li><details><summary><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1175084/testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewContentsSectionDisplayContextTest">com.liferay.site.cms.site.initializer.internal.display.context.test.ViewContentsSectionDisplayContextTest</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/262391185/history">Failed 9 of last 17</a></summary><div><ol><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1175084//testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewContentsSectionDisplayContextTest/testGetFDSActionDropdownItems">testGetFDSActionDropdownItems</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1175084/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1175084//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/13478">test-portal-acceptance-pullrequest(master)#13478</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/13478/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>cms<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/oliv-yu/liferay-portal/pull/106">oliv-yu#106</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/276286883">[master] ci:test:cms - oliv-yu &gt; oliv-yu - PR#106 - 2025-08-01[18:16:25]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>276286883</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/13478/">test-portal-acceptance-pullrequest(master)#13478</a><br/></div></div>

ak-ragnor committed:
LPD-62168 Add url and description to the json

oliv-yu committed:
LPD-49706 LPD-50637 site-cms-site-initializer: Fix some jest tests with mocks

oliv-yu committed:
LPD-49706 LPD-50637 site-cms-site-initializer: Updates after cmsGroupId was added

ak-ragnor committed:
LPD-62168 Add Test

jkappler committed:
LPD-62256 We don't allow to change parent folders in patch. We are going to create moving endpoints for this, so remove the tests

ambrinchaudhary committed:
LPD-49706 Update integration tests

rafaellajordao committed:
LPD-60567 Display RangeSelector in the CMS Performance component

rafaellajordao committed:
LPD-60567 Adapts the path of the RangeSelectorsDropdown component

rafaellajordao committed:
LPD-60567 Fix RangeSelectorsDropdown component tests

interaminense committed:
LPD-60567 Follow Up, remove unused lang and add missing ones

interaminense committed:
LPD-60567 buildLang

ambrinchaudhary committed:
LPD-58720 Avoid compilation issue. This can be moved to TS in a diff issue

marcosapmf committed:
LPD-62210 check if contextFilterString is null earlier

pyoo47 committed:
LRCI-6178 StringBuilder.isEmpty() doesn't exist prior to JDK 15

ChrisKian committed:
LPD-56434 Partially revert the commit 'LPD-56434 Simplify 28338b5ed', otherwise Group and User ResourceTest classes will fail

ambrinchaudhary committed:
LPD-58720 Update tsconfig.json file

victorg1991 committed:
LPD-61783 baseline

victorg1991 committed:
LPD-61783 Fix tests

ambrinchaudhary committed:
LPD-62332 Adds new property fixing the tests

adolfopa committed:
LPD-61564 Keep backwards compatibility

jkappler committed:
LPD-62335 Unlocalized element has only one locale, so update test

marcoscv-work committed:
LPD-62270 - Add New Space Responsive Version Step 2

marcoscv-work committed:
LPD-62270 - Remove Unnecessary CSS

Mikellorza committed:
LPD-62098 Add delete entry version confirmation modal

lfbesada committed:
LPD-61906 Implement the default method

marcoscv-work committed:
LPD-62330 - Add Temporal Autocomplete Append

marcoscv-work committed:
LPD-62270 - New Image Step 2

lfbesada committed:
LPD-61906 Add a new method to get the info item but receiving the current scope group ID as a parameter

victorg1991 committed:
LPD-62351 Fix test

Now that editable values are formatted

lfbesada committed:
LPD-61906 Baseline

jkappler committed:
LPD-62332 Instead of DepotEntry, listen for Group, since in tests one can be deleted before the other

ambrinchaudhary committed:
LPD-49706 Improve messages

alessiorendina committed:
LPD-62367 Fixed radio option behavior while editing cart item

victorg1991 committed:
LPD-62371 baseline

BryanEngler committed:
LPD-50745 Update param names

https://liferay.atlassian.net/browse/LPD-50745

rafaprax committed:
LPD-56434 SF

JhossephG committed:
LPD-61580 fix testGetOpenAPI

jkappler committed:
LPD-61967 Just throw the error and let the catch take care of it, otherwise it will be shown twice

jkappler committed:
LPD-61967 If the status is 204 return an empty string, since it means there is no body involved

rafaprax committed:
LPD-61377 SF

ryanschuhler committed:
LRSD-10044 Fix cp button link mapping

ryanschuhler committed:
LRSD-10044 Add conditional container fragments

abfranca committed:
LRSD-10158 Move sort logic to frontend

ryanschuhler committed:
LRSD-9453 Add alternate article templates

abfranca committed:
LRSD-10158 Add default icon for menu item

brianikim committed:
LPD-62187 Fix affected test

ryanschuhler committed:
LRSD-10122 Add search blueprints

ryanschuhler committed:
LRSD-10122 Fix deselect function

ryanschuhler committed:
LRSD-10122 Fix item sort

carolmariaabb committed:
LPD-62395 Adjust integration test to expose issue

To update the object definition's title field, call the existing method with the ID. This will fetch a new entity, and the previously published version will no longer have the updated title field.

carolmariaabb committed:
LPD-62395 Retrieve the updated title field, since its value may be changed after the object definition has been published

brianchandotcom committed:
LPD-60753 Fix test while keeping the SF

brianchandotcom committed:
LRSD-10430 Same thing?

brianchandotcom committed:
LRCI-6178 prep next

brianchandotcom committed:
LPD-62098 Wordsmith

brianchandotcom committed:
LPD-62098 Regen

brianchandotcom committed:
LPD-62332 Can this ever be null?

brianchandotcom committed:
LPD-50745 Sort

brianchandotcom committed:
LPD-61783 Sort

brianchandotcom committed:
LPD-61783 SF

brianchandotcom committed:
LPD-61783 SimpleDateFormatter is not thread safe

brianchandotcom committed:
LPD-61783 Rename

brianchandotcom committed:
LPD-61783 Auto SF

brianchandotcom committed:
LPD-61564 Rename

brianchandotcom committed:
LPD-61564 prep next

brianchandotcom committed:
LPD-61564 prep next

brianchandotcom committed:
LPD-62168 SF, a bit much on my part.. but future proofing

brianchandotcom committed:
LPD-61580 SF

victorg1991 committed:
LPD-61783 Sort

ambrinchaudhary committed:
LPD-49706 buildLang

oliv-yu committed:
LPD-49706 Reorder action dropdown items
